services/brig/src/Brig/User/Search/Index.hs

```diff
@@ -275,6 +275,12 @@ createIndex' failIfExists settings shardCount = liftIndexIO $ do
     throwM (IndexError "Index already exists.")
   unless ex $ do
     let fullSettings = settings ++ [ES.AnalysisSetting analysisSettings]
+    let templateName = ES.TemplateName "directory"
+    tExists <- ES.templateExists templateName
```

Comment:
It seems dangerous to hardcode the name. Can you check how to the index was added in the first place and add removal code there? Maybe there was some option to name it so we could use that instead.

Code like this should also have a limit as to how long it should be here for. We should add a comment to remind us to delete it after some point (in Jan 2022?).

Reply:
I think I found the latest reference to this template here: https://github.com/zinfra/hegemony/pull/1809